lesson_19/page_objects/apparel_shoes_page.py

```diff
@@ -0,0 +1,14 @@
+from selenium.webdriver.common.by import By
+
+from ..page_objects.base_page import BasePage
+
+
+class ApparelShoesPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    __page_title = (By.XPATH, "//div[@class='page-title']//h1")
+
+    def get_page_title(self):
+        title = self._wait_element(self.__page_title)
+        return title.text
```

Comment:
why do you wait here? just incapsulate this in get_text function in BasePage

Please fix all around the project

Reply:
get_page_title method is transferred to the BasePage. All pages were refactored.